publiq-ui/pub-input.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="input-container">
+    <pub-label v-if="label" id="form-input">{{ label }}</pub-label>
+    <b-form-input
+      id="form-input"
```

Comment:
Will this create issues when we render this component multiple times on the page?